### ISSUE TITLE
react-addons-css-transition-group

### ISSUE THREAD

**Issue by jgillich** [developit/preact-compat#310]
Using [auth0-lock](https://github.com/auth0/lock) in my Preact app, I ran into an issue that seems to be due to caused by react-addons-css-transition-group:

```
ERROR in ./~/react-addons-css-transition-group/index.js
Module not found: Error: Can't resolve 'react/lib/ReactCSSTransitionGroup' in '.../node_modules/react-addons-css-transition-group'
 @ ./~/react-addons-css-transition-group/index.js 1:17-61
 @ ./~/auth0-lock/lib/ui/box/chrome.js
 @ ./~/auth0-lock/lib/ui/box/container.js
 @ ./~/auth0-lock/lib/core/actions.js
 @ ./~/auth0-lock/lib/core.js
 @ ./~/auth0-lock/lib/index.js
 @ ./app/utils/AuthService.js
 @ ./app/components/App.js
 @ ./app/index.js
 @ multi (webpack)-dev-server/client?http://localhost:8080 ./app/index.js
```

Using webpack with the alias. Let me know if you want me to yell at the auth0 people instead for using something as big as React in their little library. 😄

**Comment by developit** [comment]
Quick fix: `react-addons-css-transition-group` to `preact-css-transition-group`.

We should add that alias to preact-compat.

**Comment by developit** [comment]
This was fixed in #186. Let me know if it's still an issue!
